.flake8

```diff
@@ -22,4 +22,9 @@ select = C,D,E,F,W,B,B950
 # operators.  It no longer agrees with PEP8.  See, for example, here:
 # https://github.com/ambv/black/issues/21. Guido agrees here:
 # https://github.com/python/peps/commit/c59c4376ad233a62ca4b3a6060c81368bd21e85b.
-ignore = E501,W503
+#
+# Also ignore flake8's error about whitespaces before a ':'. (E203)
+# No longer complies with PEP8 See example, here:
+# https://github.com/PyCQA/pycodestyle/issues/373
+# and here: https://github.com/psf/black/issues/315
+ignore = E203,E501,E712,W503
```

Comment:
The comment you wrote explains why you added E203 to the list of ignored rules, but it does not explain why you added [E712](https://www.flake8rules.com/rules/E712.html).

Reply:
Addressed here [ccbeaac](https://github.com/cisagov/pe-source/pull/1/commits/ccbeaacc8ee8b5e59a624b48b86fa721399744d4). E712 isn't necessary for this repo so removed.